pkg/ext/apiserver_suite_test.go

```diff
@@ -43,6 +219,7 @@ func (s *ExtensionAPIServerSuite) TearDownSuite() {
 	s.cancel()
 	err := s.testEnv.Stop()
 	s.Require().NoError(err)
+	os.RemoveAll(s.certTempPath)
```

Comment:
If this fails to stop, will the `certTempPath` be left around?

Maybe `s.T().TempDir()`?